script.js

```diff
@@ -0,0 +1,21 @@
+import http from 'k6/http';
+import { Rate } from 'k6/metrics';
+
+const myFailRate = new Rate('failed requests');
+
+export let options = {
+  stages: [
+    { duration: '3m', target: 100, rps: 100 },
+    { duration: '3m', target: 500, rps: 500 },
+    { duration: '3m', target: 1000, rps: 1000 }
+  ],
+  thresholds: {
+    'failed requests': ['rate<0.1']
+  }
+};
+
+export default function () {
+  const productId = Math.floor(Math.random() * 10000000);
```

Comment:
It could be beneficial to focus on testing data at higher indexes in your database. I'm testing on indexes between 9M-10M so that I can ensure that my full database is seeded and operating appropriately and not just the first few million.